python-sdk/src/langwatch/experiment/experiment.py

```
@@ -399,6 +400,184 @@ def _display_df(df: pd.DataFrame) -> None:
+        from langwatch.experiment.platform_run import (
+            EvaluatorStats,
+            ExperimentRunResult,
+            ExperimentRunSummary,
+            TargetStats,
+        )
+
+        run_url = self._run_url or ""
+
+        try:
+            df = self.results
+        except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)